Reference.md

```diff
@@ -3011,8 +3011,11 @@ There are a lot of different moves that involve cluing trash. [Here is a handy f
 
 ## Level 17 - Rarely Used Strategies - Ejection
 
-- *Ejection* refers to a move like a *Bluff*, but instead of playing an unrelated card from *Finesse Position*, someone plays an unrelated card from their *Second Finesse Position*. For example, if a player's hand is completely unclued, their *Second Finesse Position* is slot 2.
-- Just like a *Bluff*, an *Ejection* can only be performed on the very next player.
+- *Ejection* refers to a type of move like a *Bluff*, but instead of playing an unrelated card from *Finesse Position*, someone plays an unrelated card from their *Second Finesse Position*. For example, if a player's hand is completely unclued, their *Second Finesse Position* is slot 2.
+- *Ejections* arise when playing *Finesse Position* would cause *Information Desynchronization* about the initial clue. Thus, playing *Second Finesse Position* serves as a unique way to correctly *‘Shift’* the signal of certain clues.
+- This *Signal Shift* can take a variety of forms: *Play* —-> *Save* (*5 Color Ejection*), *Save* -—> *Play* (*Rank Choice Ejection*), *Save* —-> *Trash* (*Bad Chop Move Ejection*), and *Fix* —-> *Play* (*Double Play Ejection*).
```

Comment:
in this line we are inundated with every signal shift.
do we need to know that information right now?
is it better if we move this information into each section?

Reply:
Yeah I can just add a quick note under each ejection/discharge instead - sounds good